### ISSUE TITLE
bug in markdown syntax highlighter

### ISSUE THREAD

**Issue by durka** [rust-lang/crates.io#1468]
The highlighter seems to give keywords a higher precedence than strings. Take a look at the code block under "repositories" [here](https://crates.io/crates/hubcaps):

![badly highlighted code](https://alexburka.com/tmp/Screen%20Shot%202018-07-31%20at%209.41.40%20PM.png)

The source is:

```markdown
    ```rust
    let repo = github.repo("user", "repo");
    ```
```

But it seems to be highlighting `use` as a keyword instead of `"user"` as a string.

**Comment by carols10cents** [comment]
This should be reported with [PrismJS](https://github.com/PrismJS/prism) instead, because we use that to do the syntax highlighting. Thanks!

**Comment by durka** [comment]
Will do!

**Comment by durka** [comment]
@carols10cents would you mind reopening this issue? The fix is just to update PrismJS.

**Comment by durka** [comment]
So, PrismJS is a transitive dependency through ember-prism, but even the latest version only requires PrismJS 1.8.3, which is too old (this bug was fixed in 1.14.0). How do you bump a transitive dep in NPM?
